indexing-service/src/main/java/org/apache/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -553,11 +553,33 @@ public boolean isWorkerRunningTask(ZkWorker worker, String taskId)
    * Finds the worker running the task and forwards the shutdown signal to the worker.
    *
    * @param taskId - task id to shutdown
+   * @param reasonFormat - a StringUtils reason string for shutdown
+   * @param reasonArgs - args will be used during constructio of reason log line
+   */
+  @Override
+  public void shutdown(final String taskId, String reasonFormat, Object... reasonArgs)
+  {
+    if (log.isInfoEnabled()) {
```

Comment:
no need to check if isInfoEnabled, because the `log.info` internally checks it.

Reply:
Checking explicitly avoids the extra `+` string concatenation in the case that info() is not enabled. This is minor, but while we are here...